src/packages/cli/package.json

```diff
@@ -27,7 +27,7 @@
     "@kibalabs/core": "^0.5.3",
```

Comment:
i cant see if you have deleted the dialog and messagedialog from anvas. if you havent pleaase do.

Reply:
you mean the dialog files in everypage

Reply:
ye